validator/java/Code-of-Conduct.md

```diff
@@ -0,0 +1,64 @@
+# Oath Open Source Code of Conduct
```

Comment:
Hi @GeorgeLuo just a headsup that all AMP Project repos are governed by a unified Code of Conduct as you can see here: https://github.com/ampproject/meta/blob/master/CODE_OF_CONDUCT.md

I was wondering if you were adding this Code of Conduct because the existing one didn't meet your needs? Or because your company policy needs this Code of Conduct to be followed?

Reply:
@nainar we don't have any stipulation, do we need to copy your CODE_OF_CONDUCT file into our subdirectory, or if we do not include one is it implicit? Thank you

Reply:
It is implied that all conversations in all repos of the AMP Project adhere to the CoC. It is mentioned in every project like this one as well. So you should be good to go. :)